Sources/Apodini/Semantic Model Builder/OpenAPI/OpenAPIInterfaceExporter.swift

```diff
@@ -27,20 +27,27 @@ class OpenAPIInterfaceExporter: StaticInterfaceExporter {
 
     func finishedExporting(_ webService: WebServiceModel) {
         serveSpecification()
+        app.storage.set(OpenAPIStorageKey.self, to: documentBuilder.build())
     }
 
     private func serveSpecification() {
-        if let outputRoute = configuration.outputEndpoint {
-            switch configuration.outputFormat {
-            case .JSON:
-                app.vapor.app.get(outputRoute.pathComponents) { _ -> String in
-                    guard let jsonDescription = self.documentBuilder.jsonDescription else {
-                        throw Abort(.internalServerError)
-                    }
-                    return jsonDescription
+        if let output = try? self.documentBuilder.document.output(self.configuration.outputFormat) {
+            // register OpenAPI endpoint
+            app.vapor.app.get(configuration.outputEndpoint.pathComponents) { _ -> String in
+                output
+            }
+            
+            // register swagger UI endpoint
+            app.vapor.app.get(configuration.swaggerUiEndpoint.pathComponents) {_ -> Vapor.Response in
```

Comment:
Please add a space before the wildcard parameter here:
```suggestion
            app.vapor.app.get(configuration.swaggerUiEndpoint.pathComponents) { _ -> Vapor.Response in
```

Reply:
better than the swift linter 🔍 , solved with cecb08d